services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/design-kits.js

```diff
@@ -47,13 +71,76 @@ const DesignKits = ({ libraryData, navData }) => {
     )
   }
 
+  const noDesignKits = (
+    <>
+      <PageDescription>This library contains no compatible design kits.</PageDescription>
+      <P>
+        It looks like there are no design kits compatible with this library. Explore the design kit
+        catalog to find more design kits or contact libray maintainers.
+      </P>
+      <CardGroup>
+        <Column lg={4} md={4}>
+          <ResourceCard subTitle="Design kit catalog" href="/design-kits">
+            <RulerAlt size={32} />
+          </ResourceCard>
+        </Column>
+      </CardGroup>
+    </>
+  )
+
   return (
     <>
       <NextSeo {...seo} />
       <PageHeader bgColor={pageHeader?.bgColor} title="Design kits" pictogram={pageHeader?.icon} />
       <Grid>
         <Column sm={4} md={8} lg={12}>
-          <div className={pageStyles.content}>Coming soon!</div>
+          <div className={pageStyles.content}>
+            {!designKits && noDesignKits}
+            {designKits && (
+              <>
+                <PageDescription>
+                  The following design kits are compatible with the {name} library. If you are new
+                  to Carbon, check out the{' '}
+                  <Link href="/designing/get-started" passHref>
+                    <CarbonLink>design guidance</CarbonLink>
+                  </Link>{' '}
+                  to get started. You can also view all design kits in the{' '}
+                  <Link href="/design-kits" passHref>
+                    <CarbonLink>catalog</CarbonLink>
+                  </Link>
+                  .
+                </PageDescription>
+                <AnchorLinks>
+                  {filteredDesignTools.length > 2 &&
+                    filteredDesignTools.map((item, i) => (
+                      <AnchorLink key={i}>{get(designTools, `[${item}].name`)}</AnchorLink>
```

Comment:
I think we agreed not to use `lodash.get`? no real preference here 🤷‍♀️ 
```suggestion
                      <AnchorLink key={i}>{designTools?.[item]?.name}</AnchorLink>
```